core/script.runtime/src/kotlin/script/extensions/samWithReceiver.kt

```diff
@@ -16,6 +16,11 @@
 
 package kotlin.script.extensions
 
+// discuss
+
+// Is this an appropriate place to put this class?
```

Comment:
Yes, I think once KT-19120 is implemented, users can just pass something like `-Xplugin:kotlin-sam-with-receiver.jar -P plugin:sam-with-receiver:annotation= org.gradle.api.HasImplicitReceiver`